server/services/mappers/inPrisonCourseMapper.ts

```diff
@@ -0,0 +1,61 @@
+import { LearnerEducation } from '../../data/interfaces/curiousApi/LearnerEducation'
+import { InPrisonCourse } from '../interfaces/curiousService/CuriousInPrisonCourses'
+
+const toInPrisonCourse = (apiLearnerEducation: LearnerEducation): InPrisonCourse => {
+  return {
+    prisonId: apiLearnerEducation.establishmentId,
+    prisonName: apiLearnerEducation.establishmentName,
+    courseCode: apiLearnerEducation.courseCode,
+    courseName: apiLearnerEducation.courseName,
+    courseStartDate: new Date(apiLearnerEducation.learningStartDate),
+    courseStatus: toCourseStatus(apiLearnerEducation.completionStatus),
+    courseCompletionDate: apiLearnerEducation.learningActualEndDate
+      ? new Date(apiLearnerEducation.learningActualEndDate)
+      : null,
+    coursePlannedEndDate: apiLearnerEducation.learningPlannedEndDate
+      ? new Date(apiLearnerEducation.learningPlannedEndDate)
+      : null,
+    isAccredited: apiLearnerEducation.isAccredited,
+    grade: apiLearnerEducation.outcomeGrade || apiLearnerEducation.outcome || null,
+    withdrawalReason: apiLearnerEducation.prisonWithdrawalReason,
+    source: 'CURIOUS',
+  }
+}
+
+/**
```

Comment:
The comments on this method are significant and important, as they describe the status fields in the Curious API response data and how we map that into a status